capstone/src/main/java/com/google/univiz/scorecard/ScorecardConverter.java

```diff
@@ -0,0 +1,42 @@
+package com.google.univiz.scorecard;
+
+import com.google.common.base.Converter;
+import com.google.univiz.CarnegieSizeDegree;
+import com.google.univiz.CollegeData;
+import com.google.univiz.CollegeId;
+
+final class ScorecardConverter extends Converter<ScorecardData, CollegeData> {
+
+  @Override
+  protected CollegeData doForward(ScorecardData scorecardCollege) {
+    CollegeData.Builder collegeBuilder = CollegeData.builder();
```

Comment:
nit - this builder isn't used until the final .build() invocation. Just create the builder inline on line 20.